@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -439,6 +439,32 @@ export enum PoiStackMode {
     ShowParent = "show-parent"
 }
 
+/**
+ * Defines options (tokens) supported for text placements defined via [[placements]] attribute.
+ *
+ * Possible values are defined as vertical placement letter and horizontal letter, where
+ * one of the axis may be ignored and then assumed centered. Moving clock-wise, we have:
+ * `TL` (top-left), `T` (top-center), `TR` (top-right), `R` (center-right), `BR` (bottom-right),
+ * `B` (bottom-center), `BL` (bottom-left), `L` (left), `C` (center-center).
+ * Additionally instead of `T`, `B`, `L`, `R` geographic directions may be used accordingly:
+ * `N` (north), `S` (south), `E` (east), `W` (west).
+ */
+export enum PlacementToken {
+    TopLeft = "TL",
+    TopCenter = "T",
+    TopRight = "TR",
+    CenterRight = "R",
+    BottomRight = "BR",
+    BottomCenter = "B",
+    BottomLeft = "BL",
+    CenterLeft = "L",
+    CenterCenter = "C",
+    North = "N",
+    South = "S",
+    East = "E",
+    West = "W"
```

Comment:
I guess you added these to match the behaviour in native. But then you're missing the combinations NE, NW, SE, SW. Native implementation supports those too.

Reply:
Ok will add this too.